client/src/containers/overview/header/parameters/index.tsx

```diff
@@ -96,7 +98,9 @@ export default function ParametersProjects() {
   };
 
   return (
-    <div className="flex items-center space-x-4">
+    <div
+      className={`flex items-center space-x-4 2xl:flex ${filtersOpen ? "hidden 2xl:flex" : "flex"}`}
```

Comment:
Being picky here, but I think it's more readable if you use the object form here. But it's fine.